java/code/src/com/suse/manager/webui/controllers/maintenance/MaintenanceScheduleController.java

```diff
@@ -115,6 +118,22 @@ public static ModelAndView maintenanceSchedules(Request request, Response respon
         return new ModelAndView(params, "templates/schedule/maintenance-windows.jade");
     }
 
+    /**
+     * Handler for the SSM schedule assignment page.
+     *
+     * @param request the request object
+     * @param response the response object
+     * @param user the current user
+     * @return the ModelAndView object to render the page
+     */
+    public static ModelAndView ssmSchedules(Request request, Response response, User user) {
```

Comment:
I wonder whether we should move this one to the `SsmController` instead. At least that's where I would expect this to be found.

Reply:
SSM is generally structured as a collection of operations from various domains and every domain adds it's own functionality from it's own namespace. For example: `ssm/highstate` is served from `MinionController`.